lib/styles/settings/typography/_variables.scss

```diff
@@ -0,0 +1,16 @@
+:root {
+  --typography-font-family: 'Lato';
```

Comment:
if we really need this we should define full `font-family` string with fallback fonts. Right know it's possible that browser will render serif font while font is loading

Reply:
True. It's a such big think like change default font i whole css, so I think it can be manage by our styles, not from JSON.